test/helper.rb

```diff
@@ -93,6 +93,22 @@ def temp_dir(*subdirs)
   end
 end
 
+module Jekyll
+  class ModifiedReader < Reader
+    def read_directories(dir = "")
+      if dir.start_with?("/symlink") && Utils::Platforms.really_windows?
```

Comment:
Is this a bug in Jekyll that's never worked properly on Windows? Or, since symlinks aren't a thing, we don't need to be worried about it? Do you happen to know how Git on Windows handles cloning a repository with symlinks?

Reply:
Its **not a bug in Jekyll**, but some discrepancy elsewhere in Windows.
My local Windows 11 system doesn't honor the symlinks in the `test/source` directory. But now that you mentioned it, I'm not sure if it's a cloning issue or system issue because the **`windows-latest`** image on GitHub Actions do acknowledge those symlinks just like the Ubuntu image.

Reply:
Ok! Thanks for checking and adding the necessary code.